plugins/scaffolder/api-report.md

```diff
@@ -228,36 +277,51 @@ export const nextSelectedTemplateRouteRef: SubRouteRef<
 // @public
 export const OwnedEntityPickerFieldExtension: FieldExtensionComponent<
   string,
-  OwnedEntityPickerUiOptions
+  {
+    defaultKind?: string | undefined;
+    defaultNamespace?: string | false | undefined;
+    allowedKinds?: string[] | undefined;
+    allowArbitraryValues?: boolean | undefined;
+  }
+>;
+
+// @public (undocumented)
+export const OwnedEntityPickerFieldSchema: FieldSchema<
+  string,
+  {
+    defaultKind?: string | undefined;
+    defaultNamespace?: string | false | undefined;
+    allowedKinds?: string[] | undefined;
+    allowArbitraryValues?: boolean | undefined;
+  }
 >;
 
 // @public
-export interface OwnedEntityPickerUiOptions {
-  // (undocumented)
-  allowArbitraryValues?: boolean;
-  // (undocumented)
-  allowedKinds?: string[];
-  // (undocumented)
-  defaultKind?: string;
-  // (undocumented)
-  defaultNamespace?: string | false;
-}
+export type OwnedEntityPickerUiOptions =
+  typeof OwnedEntityPickerFieldSchema.uiOptionsType;
```

Comment:
Hmm, we don't particularly need the `OwnedEntityPickerUiOptions` right? Just a backwards compatibility thing? I wonder if we should just leave the existing ones in place and deprecate them? That way we can avoid the new `uiOptionsType` prop. Then again having `uiOptionsType` really doesn't hurt much either.

Reply:
Yeah, I added these just to keep the public API surface area the same as before.